dlib/CMakeLists.txt

```diff
@@ -412,6 +411,7 @@ if (NOT TARGET dlib)
             gui_widgets/drawable.cpp
             gui_widgets/canvas_drawing.cpp
             gui_widgets/style.cpp
+            gui_widgets/fonts.cpp
```

Comment:
This was moved out of here in https://github.com/davisking/dlib/pull/2273 so that the font drawing could be used with draw_string().  Although it shouldn't have gone to the top since that's for only ISO standard C++ stuff.  It should go up at https://github.com/davisking/dlib/pull/2384/files#diff-49efe726bb1d239c7d05dd384d1373297dfac8392bd41748eb5e45fd618702ffR289.  Adding it up there works for what you are doing right?

Reply:
I think 99% of the dlib users don't use the ISO only option, but in my case I wanted to use only the base64 and random functions.

If I put back the `gui_widgets/fonts.cpp` here, I guess it would break #2273 ?

I'll create another pull request so that it would not break the previous commit.

Reply:
Yep, exactly